### PR TITLE
Changed Member List number in Guilds

### PR DESCRIPTION
**Pull request by shadimoustafa**
- **Description**
When in Discover Guilds, the member list number on the left side does not fit in the box if it's over 3 digits.

- **Changes**

Before:
before_change

After:
after_change

----
UUID: 3b853dc8-4a8a-4992-a486-f2e961c71bb4

### CODE REVIEW

**Comment by negue** [comment]
@shadimoustafa Thanks for your PR, but could you revert the changes in `README.MD` and `habitica2.bat`

**Comment by negue** [comment]
@Tressley what do you think about the size changes ?

**Comment by shadimoustafa** [comment]
@negue I have manually reverted the changes in README.MD and habitica2.bat. Thanks!

**Comment by Tressley** [comment]
@negue -- LGTM. 👍

**Comment by negue** [comment]
@shadimoustafa could you remove the `habitica2.bat`since we don't need an empty batch file ^^

**Comment by shadimoustafa** [comment]
@negue  done!

File: README.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ For an introduction to the technologies used and how the software is organized,
 
 To set up a local install of Habitica for development and testing on various platforms, see [Setting up Habitica Locally](http://habitica.wikia.com/wiki/Setting_up_Habitica_Locally).
 
+Team JaMiCaSh - Jason, Michelle, Cameron, Shadi
```
Comment (paglias) [inline README.md]: can you remove this line?

**Review by paglias — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shadimoustafa) [reply README.md]: Sure,  done.!

**Comment by paglias** [comment]
@shadimoustafa  thanks for this! I'm merging the PR and awarding your contributor tier.

Note that further tiers require increasing amounts of work from one to the next, so Tier 2 may require a couple of PRs or a larger PR to attain. But keep helping out and we'll express our gratitude accordingly!

**Pull request merged**